pkgs/development/python-modules/asyncpraw/default.nix

```diff
@@ -0,0 +1,76 @@
+{
+  lib,
+  aiofiles,
+  aiohttp,
+  aiosqlite,
+  asyncprawcore,
+  buildPythonPackage,
+  fetchFromGitHub,
+  flit-core,
+  mock,
+  pytestCheckHook,
+  pytest-asyncio,
+  pytest-vcr,
+  pythonOlder,
+  requests-toolbelt,
+  update-checker,
+  vcrpy,
+}:
+
+buildPythonPackage rec {
+  pname = "asyncpraw";
+  version = "7.8.1";
+  pyproject = true;
+
+  disabled = pythonOlder "3.8";
+
+  src = fetchFromGitHub {
+    owner = "praw-dev";
+    repo = "asyncpraw";
+    tag = "v${version}";
+    hash = "sha256-glWAQoUjMFbjU3C4+MGuRGSGJS9mun15+6udMPCf9nU=";
+  };
+
+  patches = [
+    ./relax_aiosqlite_version.patch
+  ];
```

Comment:
you can and should use `pythonRelaxDeps` for this instead

Reply:
Done, TIL.